src/main/java/com/minecolonies/tileentities/TileEntityColonyBuilding.java

```diff
@@ -78,14 +78,12 @@ public void update()
     {
         super.update();
 
+        /*
         if (!worldObj.isRemote && colonyId == 0)
         {
-            Log.getLogger().fatal(String.format("TileEntityColonyBuilding at %s:[%d,%d,%d] has no colonyId",
-              worldObj.getWorldInfo().getWorldName(),
-              pos.getX(),
-              pos.getY(),
-              pos.getZ()));
+            //todo: actually do something about it and not spam the server
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)